docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Migrating from Prometheus
+weight: 320
+---
+
+# Migrating from Prometheus
+
+Migration to Grafana Agent flow mode from [Prometheus][] can be done using
+built in Grafana Agent tooling.
+
+This topic describes how to:
+
+* Convert a Prometheus configuration to a flow configuration
+* Run a Prometheus configuration natively using Grafana Agent flow mode
+
+[Prometheus]: https://prometheus.io/docs/prometheus/latest/configuration/configuration/
+
+## Components used in this topic
+
+* [prometheus.scrape][]
+* [prometheus.remote_write][]
+
+[prometheus.scrape]: {{< relref "../reference/components/prometheus.scrape.md" >}}
+[prometheus.remote_write]: {{< relref "../reference/components/prometheus.remote_write.md" >}}
+
+## Before you begin
+
+* Have an existing Prometheus configuration.
+* Have a set of Prometheus applications ready to push telemetry data to
+  Grafana Agent.
+* Be familiar with the concept of [Components][] in Grafana Agent flow mode.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+[convert]: {{< relref "../reference/cli/convert.md" >}}
+[run]: {{< relref "../reference/cli/run.md" >}}
+[Start the agent]: {{< relref "../setup/start-agent.md" >}}
+[Flow Debugging]: {{< relref "../monitoring/debugging.md" >}}
+[debugging]: #debugging
+
+## Convert a Prometheus configuration
+
+In order to fully migrate your configuration from [Prometheus] to Grafana Agent
+Flow, your Prometheus configuration must be converted into a Grafana Agent flow
+mode configuration. This will enable you to modify it going forward as a flow
+configuration and take full advantage of the many additional features available
+in Grafana Agent flow mode.
+
+> In this task, we will use the [convert][] CLI command to output flow
+> configuration from a Prometheus configuration.
+
+1. Execute the following:
+
+    ```bash
+    grafana-agent convert --format=prometheus --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+    ```
+  
+    1. Replace `INPUT_CONFIG_PATH` with the full path to the Prometheus configuration.
+    2. Replace `OUTPUT_CONFIG_PATH` with the full path to output the flow configuration.
```

Comment:
```suggestion
    Replace the following: 
      * `INPUT_CONFIG_PATH`: The full path to the Prometheus configuration.
      * `OUTPUT_CONFIG_PATH`: The full path to output the flow configuration.
```

Changed this to the format that we've been using for this elsewhere in the docs

Reply:
heads up, the other places in Get started don't do it this way. I do prefer the way you suggested.

Reply:
Yeah there's a lot of iteration needed to align things. That's OK... we get there in the end :-)